src/protocols/two_party_ecdsa/lindell_2017/party_one.rs

```diff
@@ -493,7 +495,19 @@ impl Signature {
         let s_tag_tag = BigInt::mod_mul(&k1_inv, &s_tag.0, &FE::q());
         let s = cmp::min(s_tag_tag.clone(), FE::q().clone() - s_tag_tag.clone());
 
-        Signature { s, r: rx }
+        /*
+         Calculate recovery id - it is not possible to compute the public key out of the signature
+         itself. Recovery id is used to enable extracting the public key uniquely.
+         1. id = R.y & 1
+         2. if (s > curve.q) id = id ^ 1
```

Comment:
should be  `curve.q /2 `

Reply:
Can you explain? I think @oleiba tested with this code signing for ETH.

Reply:
I meant the comment. the code is fine